src/Components/Intercom.js

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as Scrivito from 'scrivito';
+import Helmet from 'react-helmet';
+
+class Intercom extends React.Component {
+  static getIntercomAppId() {
```

Comment:
We're not using the pattern of static functions so far. AFAICT the pattern is putting helper functions after the component (see e.g. SchemaDotOrg).

* [ ] Please don't use static functions for better consistency.